xml/System.Threading.Channels/UnboundedChannelOptions.xml

```diff
@@ -33,7 +33,7 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Threading.Channels.UnboundedChannelOptions" /> abstract class. This class cannot be instantiated.</summary>
```

Comment:
>abstract class [](start = 118, length = 14)

I think this is not abstract class.

Reply:
Thanks for catching this. I'll remove it.

Reply:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Threading.Channels.UnboundedChannelOptions" /> abstract class.</summary>
```